contracts/test/Token.sol

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: LGPL-3.0-only
-pragma solidity >=0.6.0 <0.7.0;
+pragma solidity >=0.6.0 <0.8.0;
```

Comment:
Why not 0.9.0?

Reply:
The raise was to make it compile with at least the default compiler (`v0.7.6` in our case now). Any specific reason it should be raised until `v0.9`?

Reply:
This PR is fixing compilation for 0.8.x, so it makes sense that our contracts are compatible with it. No features in this contract would block the compilation with 0.8.x

Reply:
Makes sense. Done 👍🏾